future-apps/payroll/app/src/screens/components/EmployeeList.js

```diff
@@ -94,10 +98,36 @@ class EmployeeList extends React.Component {
   }
 }
 
-function mapStateToProps ({ employees = [], denominationToken = [] }) {
+function totalPaidThisYear (payments, accountAddress) {
+  const init = new BN(0)
+  const reducer = (acc, payment) => acc.add(new BN(payment.exchangeRate.amount))
```

Comment:
I think this should be something like:
`acc.add(payment.amount.amount / payment.exchangeRate.amount)`